manifests/r/RIA/DigiDoc4/4.2.8.89/RIA.DigiDoc4.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: RIA.DigiDoc4
+PackageVersion: 4.2.8.89
+PackageLocale: en-US
+Publisher: Riigi Infosüsteemi Amet
+PublisherUrl: https://www.ria.ee/
+PublisherSupportUrl: https://www.id.ee/en/id-help/
+#PrivacyUrl: 
+#Author: 
+PackageName: DigiDoc4 Client
+PackageUrl: https://www.id.ee/en/
+License: Proprietary
+#LicenseUrl: 
+#Copyright: 
+#CopyrightUrl: 
+ShortDescription: ID-software
```

```suggestion
ShortDescription: ID-software
```
Hmm is there a better description for this software?